lib/resty/string.lua

```diff
@@ -32,6 +32,11 @@ function _M.to_hex(s)
     return ffi_str(buf, buf_len)
 end
 
+function _M.from_hex(s)
```

Comment:
It's not good to add such a function here, we can only add a local function in the test case instead.